routes/bookRoute.js

```diff
@@ -6,18 +6,44 @@ const Book = require('../model/book');
 router.get('/books',(req,res) => {
     Book.find(function(err, bookList) {
         if(err) res.send({error: err.message})
-        console.log(bookList, '>>>>')
-        res.send(bookList)
+        title="Booklists";
+        res.render('book', {title:title,books:bookList});
+        //res.send(bookList)
     })
 });
 
 router.post('/books',(req,res) => {
     const data = req.body;
-    Book.create(data, (err,createData)=> {
+    Book.create(data, (err, createData)=> {
         if(err) res.send({error: err.message})
         res.send(createData)
     })
+
    
 });
 
+
+router.get('/frompug', (req, res) => {
+    title="My fisrt page with pug";
+    books = ['Crypto', 'DSA','TOC','NSA'];
+    body="Hello world!!"
+    res.render('book', {title: title,body:body,books:books});
+})
+
+/// Addding new things
+
+
+//  router.put('/books/edit/:id', (req,res) => {
```

Comment:
why are these code commented?
If they are required for the future, write the reason in code.